src/main/fc/rc_controls.h

```diff
@@ -59,16 +59,9 @@ typedef enum {
 } rollPitchStatus_e;
 
 typedef enum {
-    RC_SMOOTHING_OFF = 0,
-    RC_SMOOTHING_DEFAULT,
-    RC_SMOOTHING_AUTO,
-    RC_SMOOTHING_MANUAL
-} rcSmoothing_t;
-
-typedef enum {
-    RC_SMOOTHING_TYPE_INTERPOLATION,
-    RC_SMOOTHING_TYPE_FILTER
-} rcSmoothingType_e;
+    OFF,
+    ON,
+} rcSmoothingMode_e;
```

Comment:
This isn't used. Also, congratulations, you just defined a boolean as an enum. :stuck_out_tongue_closed_eyes: